openprescribing/media/js/src/chart.js

```diff
@@ -101,21 +101,29 @@ var analyseChart = {
 
     loadChart: function() {
         var _this = this;
+        var button_clicked = false;
+        if ($(this.el.submitButton).data('clicked'))
+            button_clicked = true;
         this.el.submitButton.button('reset');
         this.globalOptions.activeOption = 'items';
         this.setUpData();
         this.globalOptions.allMonths = utils.getAllMonthsInData(this.globalOptions.data.combinedData);
         this.globalOptions.activeMonth = this.globalOptions.allMonths[this.globalOptions.allMonths.length-1];
         this.globalOptions.friendly = formatters.getFriendlyNamesForChart(this.globalOptions);
+        this.hash = hashHelper.setHashParams(this.globalOptions);
+        if (button_clicked)
```

Comment:
ooh also just spotted while writing that that we need an ampersand in the second option too.